### PR TITLE
fix: config mismatch

### PR DESCRIPTION
**Pull request by jleach**
Fix an issue where some old config was being fetched form a constants file rather than the .env file.

### CODE REVIEW

File: app/src/components/BCIDView.tsx [hunk 1]
```diff
@@ -122,6 +123,8 @@ const BCIDView: React.FC = () => {
   const authenticateWithServiceCard = async (did: string): Promise<void> => {
     try {
       const url = `${Config.IAS_PORTAL_URL}/${did}`
+      console.log('*****************************************************')
```
Comment (amanji) [inline app/src/components/BCIDView.tsx]: Are we keeping these?

**Review by amanji — DISMISSED** [review]

**Review by amanji — COMMENTED** [review]
Sorry meant to just comment, not approve yet.

File: app/src/components/BCIDView.tsx (hunk 1 above)
Comment (jleach) [reply app/src/components/BCIDView.tsx]: Removed.

File: app/src/components/BCIDView.tsx [hunk 2]
```diff
@@ -30,6 +29,8 @@ const trustedLSBCCredentialIssuerRe =
 const redirectUrlTemplate = 'bcwallet://bcsc/v1/dids/<did>'
 const notBeforeDateTimeAsString = '2022-11-21T17:00:00.000Z'
 const connectionDelayInMs = 3000
+// const invitationId = '6cc22b56-fd0c-4b78-a7e4-c60c2e80e034'
```
Comment (amanji) [inline app/src/components/BCIDView.tsx]: Any chance this will be reused again?

Comment (jleach) [reply app/src/components/BCIDView.tsx]: Yes, for the test environment so I want to remember it.

File: app/src/components/BCIDView.tsx [hunk 3]
```diff
@@ -30,6 +29,8 @@ const trustedLSBCCredentialIssuerRe =
...
+const invitationId = '0644a296-8be7-403c-95da-2bfb77ee95f1'
```
Comment (cvarjao) [inline app/src/components/BCIDView.tsx]: This is fine for now but better to come from `invite.id` after calling `parseInvitation`

**Review by cvarjao — DISMISSED** [review]

File: app/src/components/BCIDView.tsx [hunk 4]
```diff
@@ -195,7 +201,11 @@ const BCIDView: React.FC = () => {
         )
       }
 
-      const record = await agent?.oob.receiveInvitation(invite!)
+      console.log('A')
```
Comment (cvarjao) [inline app/src/components/BCIDView.tsx]: just remove these `console.log` lines

Comment (jleach) [reply app/src/components/BCIDView.tsx]: Fixed.

**Review by amanji — APPROVED** [review]

**Pull request merged**